src/mcp/tools/crashlytics/issues.ts

```diff
@@ -23,9 +26,21 @@ export const get_issue = tool(
     },
   },
   async ({ appId, issueId }) => {
-    if (!appId) return mcpError(`Must specify 'appId' parameter.`);
-    if (!issueId) return mcpError(`Must specify 'issueId' parameter.`);
-
+    const result: CallToolResult = { content: [] };
+    if (!appId) {
+      result.isError = true;
+      result.content.push({ type: "text", text: "Must specify 'appId' parameter" });
+      result.content.push({ type: "text", text: forceAppIdGuide });
```

Comment:
We don't return the issue guide here. At some iteration, I feel like we were doing that. If we were, why did we pull it back?